build/images/dfdaemon/Dockerfile

```diff
@@ -29,11 +29,28 @@ RUN if [ "$(uname -m)" = "ppc64le" ]; then \
 
 FROM ${BASE_IMAGE}
 
+ARG USER_NAME=dragonfly
```

Comment:
I think the default user should be root, please do not to break compatibility with previous versions of dragonfly.

Reply:
the code only adds the none root user and group, the default user is still root.

Reply:
Please do not change the default root user using command.

Reply:
> Please do not change the default root user using command.

If I understand correctly
1. the default user when running container is still root and when can using the dragonfly by adding user argument
2. the binary would belongs to dragonfly/dragonfly can it can still work within root user.